src/fswAlgorithms/sensorInterfaces/simpleInstrumentController/simpleInstrumentController.c

```diff
@@ -84,12 +86,13 @@ void Update_simpleInstrumentController(simpleInstrumentControllerConfig *configD
 
     // Compute the norm of the attitude error
     sigma_BR_norm = v3Norm(attGuidInMsgBuffer.sigma_BR);
+    omega_BR_norm = v3Norm(attGuidInMsgBuffer.omega_BR_B);
 
     // If the target has not been imaged
     if (!configData->imaged) {
         /* If the attitude error is less than the tolerance and the groundLocation is accessible, turn on the instrument and
```

Comment:
:small_red_triangle_down: After the change to the conditional, this comment is no longer correct.

Reply:
Revised comment to reflect changes.